lib/components/artist/shows/current_shows.js

```diff
@@ -49,16 +49,14 @@ var styles = StyleSheet.create({
 
 export default Relay.createContainer(CurrentShows, {
   fragments: {
-    artist: () => Relay.QL`
-      fragment on Artist {
-        partner_shows(status: "running") {
-          meta_image {
-            cropped(width: 350, height: 200) {
-              url
-            }
+    shows: () => Relay.QL`
+      fragment on PartnerShow @relay(plural: true) {
```

Comment:
The `@relay(plural: true` part signals to Relay that we expect this prop to be a collection.